### ISSUE TITLE
Promotion type missing in GUI

### ISSUE THREAD

**Issue by MiranCZ** [SebLague/Chess-Challenge#104]
I know its not that much important, but its not possible to promote to anything other than a queen as a "human player"

**Comment by MrBrain295** [comment]
It is important to avoid stalemate sometimes.

**Comment by MrBrain295** [comment]
It is unlikely but it’s happened to me in a game once.

**Comment by eliseukloster** [comment]
I agree this should be in the GUI. My pawns are automatically promoted to Queens as the human player.
